wikimedia/wikipedia-ios
# Maybe fix AppViewController crash

**Fixes Phabricator ticket:** N/A

### Notes
* I could be talked out of this one - it's been more of a warmup task for me this morning. This is the highest non-widget crash in 6.7.0. It fails in `WMFAppViewController`'s `viewDidLoad`, opening in project points to the line `self.navigationItem.backButtonDisplayMode = UINavigationItemBackButtonDisplayModeGeneric;` with a `NSObject doesNotRecognizeSelector` failure in the stack trace. I can't reproduce it, but I noticed all of the crash samples show an iOS14 build number of either beta 1 or beta2, so my guess is that this was an Apple-side bug that has since been fixed, and these crashes will eventually go away on their own.

### Test Steps
1. Add a breakpoint at the `backButtonDisplayMode` assignment in `AppViewController` and in one of the Swift usages (I did `ArticleViewController`), run on iOS12 or iOS13. Confirm breakpoints are not hit.
2. Do the same for iOS14. Confirm breakpoints are still hit.
